dojo/db_migrations/0093_endpoint_host_migration.py

```diff
@@ -99,6 +103,44 @@ def clean_hosts(apps, schema_editor):
         raise FieldError('It is not possible to migrate database because there is/are {} broken endpoint(s). '
                          'Please check logs.'.format(len(broken_endpoints)))
 
+    to_be_deleted = set()
+    for product in Product_model.objects.all().distinct():
+        for endpoint in Endpoint_model.objects.all():
```

Comment:
Is it need to iterate over all products times all endpoints, or would it be enough to just to all endpoints and use endpoint.product for the filter?

Reply:
fixed by commit